A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/EICAS/A320_Neo_EICAS.js

```diff
@@ -87,17 +87,17 @@ class A320_Neo_EICAS extends Airliners.BaseEICAS {
         this.changePage("DOOR"); // This should get the ECAM into the "unselected" state
         this.localVarUpdater = new LocalVarUpdater();
 
-        SimVar.SetSimVarValue("LIGHT POTENTIOMETER:7","FLOAT64",0);
-        SimVar.SetSimVarValue("LIGHT POTENTIOMETER:14","FLOAT64",0);
-        SimVar.SetSimVarValue("LIGHT POTENTIOMETER:15","FLOAT64",0);
-        SimVar.SetSimVarValue("LIGHT POTENTIOMETER:16","FLOAT64",0);
-        SimVar.SetSimVarValue("LIGHT POTENTIOMETER:17","FLOAT64",0.1);
-        SimVar.SetSimVarValue("LIGHT POTENTIOMETER:18","FLOAT64",0.1);
-        SimVar.SetSimVarValue("LIGHT POTENTIOMETER:19","FLOAT64",0.1);
-        SimVar.SetSimVarValue("LIGHT POTENTIOMETER:20","FLOAT64",0.1);
-        SimVar.SetSimVarValue("LIGHT POTENTIOMETER:21","FLOAT64",0.1);
-        SimVar.SetSimVarValue("LIGHT POTENTIOMETER:22","FLOAT64",0.1);
-        SimVar.SetSimVarValue("LIGHT POTENTIOMETER:23","FLOAT64",0.1);
+        SimVar.SetSimVarValue("LIGHT POTENTIOMETER:77","FLOAT64",0);
+        SimVar.SetSimVarValue("LIGHT POTENTIOMETER:84","FLOAT64",0);
+        SimVar.SetSimVarValue("LIGHT POTENTIOMETER:85","FLOAT64",0);
+        SimVar.SetSimVarValue("LIGHT POTENTIOMETER:86","FLOAT64",0);
+        SimVar.SetSimVarValue("LIGHT POTENTIOMETER:87","FLOAT64",0.1);
+        SimVar.SetSimVarValue("LIGHT POTENTIOMETER:88","FLOAT64",0.1);
+        SimVar.SetSimVarValue("LIGHT POTENTIOMETER:89","FLOAT64",0.1);
+        SimVar.SetSimVarValue("LIGHT POTENTIOMETER:90","FLOAT64",0.1);
+        SimVar.SetSimVarValue("LIGHT POTENTIOMETER:91","FLOAT64",0.1);
+        SimVar.SetSimVarValue("LIGHT POTENTIOMETER:92","FLOAT64",0.1);
+        SimVar.SetSimVarValue("LIGHT POTENTIOMETER:93","FLOAT64",0.1);
```

Comment:
Do we really need these still?

Reply:
not sure, but i dont think that needs to be dealt with in this pr